main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Consider letting the user input a number instead of hardcoding 9 as the stopping condition in the while loop.